types/mapcmd/map.go

```diff
@@ -241,7 +241,7 @@ func decorateCommandWithMapNameFlags(cmd *cobra.Command, mapName *string, requir
 }
 
 func decorateCommandWithMapKeyTypeFlags(cmd *cobra.Command, mapKeyType *string, required bool) {
-	cmd.Flags().StringVar(mapKeyType, MapKeyTypeFlag, "string", fmt.Sprintf("type of the key, one of: %s", strings.Join(internal.SupportedTypeNames, ",")))
+	cmd.Flags().StringVar(mapKeyType, MapKeyTypeFlag, "", fmt.Sprintf("type of the key, one of: %s", strings.Join(internal.SupportedTypeNames, ",")))
```

Comment:
How about "key type" instead of "type of the key" to make it shorter?

Reply:
Good idea https://github.com/hazelcast/hazelcast-commandline-client/pull/114/commits/33e5d0a48d545a710b929043f671ab0808485bcf